docs/en/observability/logs-parse-filter.asciidoc

```diff
@@ -0,0 +1,1227 @@
+
+For example, you might want to troubleshoot an issue that happened at a specific date or time. To do this, use a {ref}/query-dsl-bool-query.html[boolean query] made up of a {ref}/query-dsl-range-query.html[range query] to filter for the specific timestamp range and a {ref}/query-dsl-term-query.html[term query] to filter for `WARN` and `ERROR` log levels.
+
+Before setting up your query, add some logs with varying timestamps and log levels to your data stream with the following command:
+
+[source,console]
+----
+POST logs-example-default/_bulk
+{ "create": {} }
+{ "message": "2023-09-15T08:15:20.234Z WARN 192.168.1.101 Disk usage exceeds 90%." }
+{ "create": {} }
+{ "message": "2023-09-14T10:30:45.789Z ERROR 192.168.1.102 Critical system failure detected." }
+{ "create": {} }
+{ "message": "2023-09-10T14:20:45.789Z ERROR 192.168.1.105 Database connection lost." }
+{ "create": {} }
+{ "message": "2023-09-20T09:40:32.345Z INFO 192.168.1.106 User logout initiated." }
+----
+
+Let's look into an event that occurred between September 14th and 15th. The following boolean query filters for logs with timestamps during those days that have a log level of `ERROR` or `WARN`.
+
+[source,console]
+----
+POST /logs-example-default/_search
+{
+  "query": {
+    "bool": {
+      "filter": [
+        {
+          "range": {
+            "@timestamp": {
+              "gte": "2023-09-14T00:00:00",
+              "lte": "2023-09-15T23:59:59"
+            }
+          }
+        },
+        {
+          "terms": {
+            "log.level": ["WARN", "ERROR"]
+          }
+        }
+      ]
+    }
+  }
+}
+----
+
+The filtered results should show `WARN` and `ERROR` logs that occurred within the timestamp range:
+
+[source,JSON]
+----
+{
+  ...
+  "hits": {
+    ...
+    "hits": [
+      {
+        "_index": ".ds-logs-example-default-2023.09.25-000001",
+        "_id": "JkwPzooBTddK4OtTQToP",
+        "_score": 0,
+        "_source": {
+          "message": "192.168.1.101 Disk usage exceeds 90%.",
+          "log": {
+            "level": "WARN"
+          },
+          "@timestamp": "2023-09-15T08:15:20.234Z"
+        }
+      },
+      {
+        "_index": ".ds-logs-example-default-2023.09.25-000001",
+        "_id": "A5YSzooBMYFrNGNwH75O",
+        "_score": 0,
+        "_source": {
+          "message": "192.168.1.102 Critical system failure detected.",
+          "log": {
+            "level": "ERROR"
+          },
+          "@timestamp": "2023-09-14T10:30:45.789Z"
+        }
+      }
+    ]
+  }
+}
+----
+
+[discrete]
+[[logs-filter-discover]]
+=== Filter logs with Discover
+
+Discover is a {kib} tool that focuses on rapid consumption of {es} data, including logs, using the {kibana-ref}/kuery-query.html[{kib} Query Language (KQL)]. Navigate to Discover from the navigation menu by clicking *Discover* under *Analytics*.
+
+[discrete]
+[[logs-filter-discover-logs-data-view]]
+==== `logs-*` data view
```

Comment:
If we go to Logs Explorer, data views are not needed anymore as we pregenerate these and users can directly start to explore. Lets change this all to Logs Explorer.